.gitlab/deploy_packages/oci.yml

```diff
@@ -23,9 +23,9 @@ include:
     - ${CI_PROJECT_DIR}/tools/ci/retry.sh git clone --depth=1 https://github.com/DataDog/datadog-packages /tmp/datadog-packages
     - cd /tmp/datadog-packages/cmd/datadog-package
     - go build .
-    - ./datadog-package push registry.ddbuild.io/ci/remote-updates/${OCI_PRODUCT}:${VERSION} ${OMNIBUS_PACKAGE_DIR}/${OCI_PRODUCT}-${MAJOR_VERSION}.*.oci.tar
```

Comment:
does this change the behavior? The intent was to match `7.*` instead of the entire version string. Was the pattern too permissive?

Reply:
Makes sense, thanks for the clarification!